routes/api.php

```diff
@@ -34,17 +34,55 @@
     Route::post('/customers/interests','App\Http\Controllers\CustomersController@actionCustomerInterest');
     Route::post('/mailNewPassword','App\Http\Controllers\CustomersController@actionForgetPassword');
 
+    //vendor profile
+    Route::post('/vendors/profiles','App\Http\Controllers\CustomersProfileController@actionVendorProfile');
     //events api
-    Route::post('/createEvent','App\Http\Controllers\EventsController@createEvent');
-    Route::get('/events/id={customerId}','App\Http\Controllers\EventsController@fetchEvents');
-    Route::post('/events/id={customerId}','App\Http\Controllers\EventsController@updateEvents');
-    Route::get('/fetchAllevents','App\Http\Controllers\EventsController@fetchAllEvents');
-    Route::post('/events/{eventId}/tickets','App\Http\Controllers\EventsController@createEventTicket');
-    Route::get('/events/{eventId}/tickets','App\Http\Controllers\EventsController@fetchEventTicket');
-    Route::post('/events/{eventId}/dress-code','App\Http\Controllers\EventsController@createEventDresscode');
-    Route::get('/events/{eventId}/dress-code','App\Http\Controllers\EventsController@fetchEventDresscode');
- 
+    Route::post('/createEvent','App\Http\Controllers\Event\EventsController@createEvent');
+    Route::get('/events/id={customerId}','App\Http\Controllers\Event\EventsController@fetchEvents');
+    Route::put('/events/id={eventId}','App\Http\Controllers\Event\EventsController@updateEvents');
+    Route::get('/fetchAllevents','App\Http\Controllers\Event\EventsController@fetchAllEvents');
+    Route::post('/publishEvent/id={eventId}','App\Http\Controllers\Event\EventsController@actionPublishEvent');
+    
+    // delete event api
+    Route::delete('/deleteAnEvent/id={eventId}','App\Http\Controllers\Event\EventsController@deleteAnEvent');
```

Comment:
Please lets use `events/{eventId}` instead of `deleteAnEvent/id`, the HTTP method already tells us it is a delete operation.